src/commands/Announcement/announcement.ts

```diff
@@ -19,6 +19,7 @@ export default class extends SkyraCommand {
 			extendedHelp: language => language.tget('COMMAND_ANNOUNCEMENT_EXTENDED'),
 			permissionLevel: PermissionLevels.Administrator,
 			requiredGuildPermissions: ['MANAGE_ROLES'],
+			requiredPermissions: ['EMBED_LINKS'],
```

Comment:
This also requires two more permissions, one to add reactions, and another to delete them:

```suggestion
			requiredPermissions: ['ADD_REACTIONS', 'MANAGE_MESSAGES', 'EMBED_LINKS'],
```

Reply:
Alright then